docs/04-operation-guides/operations/evnt-02-eventing-metrics.md

```diff
@@ -3,22 +3,23 @@ title: Kyma Eventing Metrics
 ---
 
 Kyma Eventing provides several Grafana Dashboard with various [metrics](./evnt-02-eventing-metrics.md), so you can monitor statistics and other information in real time.
+The metrics follow the [Prometheus naming convention](https://prometheus.io/docs/practices/naming/).
 
 ### Metrics Emitted by Eventing Publisher Proxy:
 
-| Metric    |  Description |
-|-------------|:--------------|
-| **event_publish_to_messaging_server_errors_total** | Total number of errors while sending events to the messaging server. |
-| **event_publish_to_messaging_server_latency** | Duration of sending events to the messaging server. |
-| **event_type_published** | Total number of event publishing requests to the messaging server for a given event type. |
-| **event_requests** | Total number of event publishing requests to the messaging server.  |
+| Metric                                                          | Description                                                                                    |
+|-----------------------------------------------------------------|:-----------------------------------------------------------------------------------------------|
+| **eventing_epp_errors_total**                                   | Total number of errors while sending events to the messaging server.                           |
+| **eventing_epp_messaging_server_latency_duration_milliseconds** | Duration of sending events to the messaging server in milliseconds.                            |
+| **nats_epp_event_type_published_total**                         | Total number of event publishing requests to the NATS messaging server for a given event type. |
+| **eventing_epp_requests_total**                                 | Total number of event publishing requests to the messaging server.                             |
 
 ### Metrics Emitted by Eventing Controller:
 
-| Metric    |  Description |
-|-------------|:--------------|
-| **event_type_subscribed** | All the eventTypes subscribed using the Subscription CRD. |
-| **delivery_per_subscription** | Number of dispatched events per subscription with information regarding the status code and its sink. |
+| Metric                                      | Description                                                                                                 |
+|---------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **nats_ec_event_type_subscribed_total**     | Total number of all the eventTypes subscribed using the Subscription CRD.                                   |
+| **nats_ec_delivery_per_subscription_total** | Total number of dispatched events per subscription with information regarding the status code and its sink. |
```

Comment:
```suggestion
| **nats_ec_delivery_per_subscription_total** | Total number of dispatched events per subscription, with status code and sink. |
```
Maybe we can shorten this without losing meaning?

Reply:
done